### ISSUE TITLE
Make compatible with node 7.x

### ISSUE THREAD

**Issue by petergoes** [voorhoede/fastatic#120]
We have, deep down, a dependency on `graceful-fs`. This module breaks the build on node 7.x. the `graceful-fs` module should be updated so builds are passing again. Then the `engines` property in the package.json can be updated or removed

**Comment by jpsc** [comment]
I don't think we can do much about this. The only way we can help on this is actually make a pull request to the module that is using it. But for example gulp uses this same module and until they release the version 4.0 there's not much we can change.
